lib/instrumentation/http-shared.js

```diff
@@ -78,7 +78,9 @@ function isRequestBlacklisted (agent, req) {
   for (i = 0; i < agent._conf.ignoreUrlRegExp.length; i++) {
     if (agent._conf.ignoreUrlRegExp[i].test(req.url)) return true
   }
-
+  for (i = 0; i < agent._conf.transactionIgnoreUrlRegExp.length; i++) {
+    if (agent._conf.transactionIgnoreUrlRegExp[i].test(req.url)) return true
+  }
```

Comment:
nit: restore the blank line before the `ua` block?

Reply:
👍